doc/sphinx-guides/source/admin/metadatacustomization.rst

```diff
@@ -593,10 +593,27 @@ Running a curl command like "load" example above should make the new custom meta
 Updating the Solr Schema
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
-Once you have enabled a new metadata block you should be able to see the new fields in the GUI but before you can save the dataset, you must add additional fields to your Solr schema. You should run the following curl command to have Dataverse output the "field name" and "copyField" elements for all the metadata fields that have been loaded into Dataverse:
+Once you have enabled a new metadata block you should be able to see the new fields in the GUI but before you can save
+the dataset, you must add additional fields to your Solr schema.
+
+An API endpoint of Dataverse provides you with a generated set of all fields that need to be added to the Solr schema
+configuration, including any enabled metadata schemas:
 
 ``curl http://localhost:8080/api/admin/index/solr/schema``
 
+For convenience and automation you can use the *updateSchemaCMB.sh* script. It downloads, parses and writes the schema
```

Comment:
In the future (not in scope for this pull request), it might be nice to walk through a fake example of adding a metadata block with a single field. We could call this new script as part of that walkthrough.